apps/website/cli/deploy.ts

```diff
@@ -39,6 +39,7 @@ export default {
         await uploadFolderToS3({
             path: buildFolderPath,
             bucket: websiteOutput.appStorage,
+            acl: "private",
```

Comment:
Files can no longer be uploaded with `public` access, or we get 403 error during upload.

Reply:
Cool.